lib/main.js

```diff
@@ -212,21 +229,18 @@ module.exports = class Router extends EventEmitter {
                 res.locals.response.file
             );
 
-            if (response.payload.length === response.response.length) {
-                return res.status(200).json(response);
-            }
-
-            res.status(202).json(response);
+            res.status(200).json(response);
```

Comment:
This is a breaking change. 

1. The block that checks payload and response lengths will never be executed since `response.response` is always (as far as I can tell) an object, not an array so no `length` property so it has been deleted.
2. Since all processing is done before the server responds, it doesn't seem right to return a `202` so this has been changed to `200`

Reply:
Indeed!

Reply:
That actually cleans up the tests a bit, nice